nextjs/nextjs-routes.d.ts

```diff
@@ -19,10 +19,13 @@ declare module "nextjs-routes" {
     | StaticRoute<"/api/csrf">
     | StaticRoute<"/api/healthz">
     | StaticRoute<"/api/media-type">
+    | StaticRoute<"/api/noves/describe">
+    | StaticRoute<"/api/noves/history">
+    | StaticRoute<"/api/noves/translate">
```

Comment:
This wasn't changed because we didn't have the new URLs from the proxy at the time this initial refactoring was done. But yes the plan was always to replace it with the new custom proxy. It will be changed.